tests/config/serverFixtures.ts

```diff
@@ -47,30 +49,9 @@ export const serverFixtures: Fixtures<ServerFixtures, ServerWorkerOptions> = {
     const httpsServer = await TestServer.createHTTPS(assetsPath, httpsPort, loopback);
     httpsServer.enableHTTPCache(cachedPath);
 
-    const socksServer = socks.createServer((info, accept, deny) => {
-      const socket = accept(true);
-      if (!socket)
-        return;
-      socket.on('data', data => {
-        if (!data.toString().includes('\r\n\r\n'))
-          return;
-        const body = '<html><title>Served by the SOCKS proxy</title></html>';
-        socket.end([
-          'HTTP/1.1 200 OK',
-          'Connection: close',
-          'Content-Type: text/html',
-          'Content-Length: ' + Buffer.byteLength(body),
-          '',
-          body
-        ].join('\r\n'));
-      });
-      // Catch and ignore ECONNRESET errors.
-      socket.on('error', () => {});
```

Comment:
I wonder whether we need something to handle this case in the new code.

Reply:
It would emit `this._socksProxy.addListener(SocksProxy.Events.SocksError` which wouldn't end up in unhandled rejections with the new implementation since its not an event with the name `"error"`.